### PR TITLE
enable coverage

### PR DESCRIPTION
**Pull request by MartinNowak**
- coverage linkage issue is resolved in 2.078.1-beta.1
- use simpler default codecov config and only disable comments

### CODE REVIEW

**Comment by MartinNowak** [comment]
I think you still need to enable codecov integration for them to post a CI status here.
https://github.com/apps/codecov

File: .travis.yml [hunk 1]
```diff
@@ -2,11 +2,10 @@ language: d
 
 matrix:
   include:
-    - d: dmd
+    # 2.078.1-beta.1 needed for coverage fix (Issue 13742)
+    - d: dmd-beta
       script:
-        # requires fix for Issue 13742
-        - dub test #-b unittest-cov
-        - dub build -b ddox
```
Comment (thaven) [inline .travis.yml]: Why is this line deleted? is it somehow not necessary anymore to build the docs?

Comment (MartinNowak) [reply .travis.yml]: My bad, it's still neeeded of course @thaven.
